packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -96,6 +96,14 @@ class ConfigManager implements IConfigManager {
 
     updateStorageState(this.logger);
 
+    let storageType = state.loadOptions.value.storage?.type;
+    if (!isValidStorageType(storageType)) {
```

Comment:
Should we also validate individual entries?

Reply:
In store manager, we will be showing a warning message